src/go/plugin/go.d/config/go.d/snmp.profiles/default/mikrotik-router.yaml

```
@@ -186,23 +186,23 @@ metrics:
         transform: |
           {{- $sensorType := index .Metric.Tags "sensor_type" | default "" -}}
           {{- $config := get (dict 
-              "1" (dict "name" "temperature" "unit" "celsius" "family" "Sensor/Temperature/Value" 
+              "1" (dict "name" "temperature" "unit" "cel" "family" "Sensor/Temperature/Value" 
                    "desc" "Temperature reading")
-              "2" (dict "name" "fan_speed" "unit" "rpm" "family" "Sensor/FanSpeed/Value" 
+              "2" (dict "name" "fan_speed" "unit" "{revolutions}/min" "family" "Sensor/FanSpeed/Value" 
```

```suggestion
              "2" (dict "name" "fan_speed" "unit" "{revolutions}/min" "family" "Sensor/FanSpeed/Value" 
```

`revolutions` or `revolution`?